spec/entities/processor/metric_result_spec.rb

```diff
@@ -93,4 +93,60 @@
       expect(subject.value).to eq(42)
     end
   end
+
+  describe 'module_result_id=' do
```

Comment:
I think this test is unnecessary since we are not overriding the module_result_id's accessor.